yarn.lock

```diff
@@ -6407,6 +6407,13 @@ blockprotocol@0.0.1:
   dependencies:
     "@types/react" "^17.0.38"
 
+blockprotocol@0.0.6:
+  version "0.0.6"
+  resolved "https://registry.yarnpkg.com/blockprotocol/-/blockprotocol-0.0.6.tgz#9a547a9889f06158634124608fba0e73da1e8cab"
+  integrity sha512-iNpPKjahDrYtct2Pg4E9QzaAMH44KWi3AGA4lUOyhi6QT1T/y7xNv+U56eKMd4xQ0zsoZtCpEMDSKhYDXUhDvQ==
+  dependencies:
+    "@types/react" "^17.0.38"
+
```

Comment:
Pulled by `mock-block-dock`. We should consider upgrading `block-protocol` in blocks: https://app.asana.com/0/0/1201833268004928/f (internal link)